cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -20,41 +20,40 @@
     <property>
         <name>root.namespace</name>
         <value>cdap</value>
-        <description>Specifies the root namespace</description>
+        <description>Namespace for this CDAP instance</description>
```

Comment:
I don't think this is sufficient. Also, this could be confused with namespaces in CDAP. This particular property is used as:
1. The parent (or root) node for all nodes that CDAP creates in ZooKeeper.
2. The directory under which all CDAP data and metadata is stored on HDFS
3. The prefix for all HBase tables that CDAP creates

Also, this value *must* be alphanumeric, otherwise CDAP will throw an error while starting up.